themes/kien-theme/docroot/_diffs/js/main.js

```diff
@@ -0,0 +1,59 @@
+AUI().ready(
+
+	/*
+	This function gets loaded when all the HTML, not including the portlets, is
+	loaded.
+	*/
+	'aui-base',
+	'event-hover',
+	'node-event-delegate',
+	'transition',
+
+	function(Y) {
+		Y.one('.child-menu').hide(true);
+		function over() {
+			this.one('.child-menu').show(true);

```

Comment:
When looking up DOM elements using ```.one()``` you should check to make sure the element exists before doing anything on it.

Ex.

```javascript
var node = A.one('.child-menu');

if (node) {
    node.show();
}